src/ui/users/common/UserForm.js

```diff
@@ -48,227 +47,236 @@ export const renderStaticField = (field) => {
   );
 };
 
-const msgs = defineMessages({
-  username: {
-    id: 'user.table.username',
-    defaultMessage: 'Username',
-  },
-  password: {
-    id: 'user.password',
-    defaultMessage: 'Password',
-  },
-  passwordConfirm: {
-    id: 'user.passwordConfirm',
-    defaultMessage: 'Confirm Password',
-  },
+const userPassCharsValid = (value, { createError, path }) => {
```

Comment:
shouldn't we place it on a validations file?

Reply:
Ah yes. Just noticed that there's now a `formikValidation.js` file. Will move that.